### PR TITLE
Enable upper case slugs

### PR DESCRIPTION
**Pull request by mwasilew**
Signed-off-by: Milosz Wasilewski <milosz.wasilewski@linaro.org>

### CODE REVIEW

**Comment by mwasilew** [comment]
Do we need a test for that?

File: squad/api/urls.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from . import ci
 
 
-slug_pattern = '[a-z0-9_.-]+'
+slug_pattern = '[A-Za-z0-9_.-]+'
```
Comment (terceiro) [inline squad/api/urls.py]: aaaah ... that's why. I think the right fix for this is to have a single definition of what's a valid slug. IMO we should keep it in `squad.core.models`, and reuse it here

File: squad/frontend/urls.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 from . import comparison
 from . import tests
 
-slug_pattern = '[a-z0-9_.-]+'
+slug_pattern = '[A-Za-z0-9_.-]+'
```
Comment (terceiro) [inline squad/frontend/urls.py]: ditto

**Comment by terceiro** [comment]
this was implemented in f8da286

**Pull request closed without merging**